src/mame/video/redalert.cpp

```diff
@@ -171,6 +235,14 @@ VIDEO_START_MEMBER(redalert_state,redalert)
 	m_control_xor = 0x00;
 }
 
+VIDEO_START_MEMBER(redalert_state,demoneye)
+{
+	VIDEO_START_CALL_MEMBER( redalert );
+	
+	save_pointer(NAME(m_demoneye_bitmap_reg), 4);
```

Comment:
a save_pointer isn't needed here, MAME can take care of arrays just fine with save_item

Reply:
Driver needs full renaming anyway, should be `m27.cpp` not `redalert.cpp`. 
Also it's prolly worth to split states into something more granular (i.e. I don't see a need to have palette defined in screen_update fn).